scripts/gulp/plugins/gulp-component-menu-behaviors.ts

```diff
@@ -64,7 +63,23 @@ export default () => {
       if (!variation.specification) {
         const variationName = variation.name.replace('.ts', '');
         const definitionName = `${variationName}Definition`;
-        const definition = behaviorDefinitions[definitionName];
+
+        const absPathToBehaviorDefinition = path.normalize(
+          path.join(
+            require.resolve(`@fluentui/a11y-testing`),
+            '../',
+            'definitions',
+            behaviorName,
+            `${definitionName}.ts`,
+          ),
+        );
+        // delete require cache only works for absolute file path
```

Comment:
```suggestion
        // delete require cache only works for absolute file path, is required to get latest changes in behaviors for watch process
```

Can we include some reasoning? Feel free to suggest a better description 🐱